pyaedt/maxwell.py

```diff
@@ -1465,20 +1523,24 @@ def enable_harmonic_force(
         Parameters
         ----------
         objects : list
-            Object list to enable force computation.
+            List of object names for force calculations.
         force_type : int, optional
-            Force Type. `0` for Objects, `1` for Surface, `2` for volumetric.
+            Force Type. ``0`` for Objects, ``1`` for Surface, ``2`` for volumetric.
         window_function : str, optional
-            Windowing function. Default is `"Rectangular"`.
+            Windowing function. Default is ``"Rectangular"``.
+            Available options are: ``"Rectangular"``, ``"Tri"``, ``"Van Hann"``, ``"Hamming"``,
+            ``"Blackman"``, ``"Lanczos"``, ``"Welch"``.
```

Comment:
```suggestion
            Windowing function. The default is ``"Rectangular"``.
            Options are:
            
            - ``"Rectangular"``
            - ``"Tri"``
            - ``"Van Hann"``
            - ``"Hamming"``
            - ``"Blackman"``
            - ``"Lanczos"``
            - ``"Welch"``
```
It's much easier to go through a bulleted list than read it in structure form. Would it perhaps be helpful to alphabetize the options?